tasty-query/shared/src/main/scala/tastyquery/Symbols.scala

```diff
@@ -150,6 +150,15 @@ object Symbols {
       case cls: ClassSymbol => cls.is(Module) && cls.owner.isStaticOwner
       case _                => false
 
+    private[Symbols] final def staticOwnerPrefix(using Context): Type = this match
+      case pkg: PackageSymbol =>
+        pkg.packageRef
+      case cls: ClassSymbol if cls.is(Module) =>
+        cls.owner.staticOwnerPrefix.select(cls.moduleValue.get)
```

Comment:
`moduleValue` still needs patching to enable non-package owners

Reply:
I pushed an additional commit that addresses that.